OpenEdXMobile/src/main/java/org/edx/mobile/third_party/crop/CropUtil.java

```diff
@@ -141,18 +143,13 @@ public static void crop(@NonNull Context context, @NonNull Uri uri, @NonNull Rec
         try {
             final OutputStream outputStream = new FileOutputStream(file);
             try {
-                croppedImage.compress(Bitmap.CompressFormat.JPEG, 90, outputStream);
+                croppedImage.compress(Bitmap.CompressFormat.JPEG, 100, outputStream);
             } finally {
                 outputStream.close();
             }
         } finally {
             croppedImage.recycle();
         }
-
-        CropUtil.setExifOrientation(
```

Comment:
`setExifOrientation` is now unused and can be deleted. And after deleting it `getExifOrientationFromRotation` can also be deleted.